db/seeds.rb

```diff
@@ -93,6 +93,11 @@ def add_allergens_and_diets(item)
 @allergens_list = ['milk', 'eggs', 'fish', 'shellfish', 'tree nuts', 'peanuts', 'wheat', 'soybeans']
 @diets_list = ['vegan', 'vegetarian', 'pescatarian', 'lactose free', 'gluten free']
 @true_or_false = [true, false]
+# Random simply ingredient descriptions
```

Comment:
to make the vegetables have a more unique item description (still random and generic though)